scripts/config/jest/index.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
```

Comment:
For example, the other script file just imports src code: https://github.com/elastic/kibana/blob/master/scripts/es_archiver.js